README.md

```diff
@@ -207,6 +207,11 @@ sfdx sgd:source:delta --from "HEAD~1" # right git shortcut with windows because
 
 ### CI/CD specificity
 
+In CI/CD pipelines, for most of the CI/CD provider, the checkout operation fetch only the last commit of the branch currently evaluated.
+This is done prbably to have the smallest network load possible and decrease clone time.
```

Comment:
I would remove this sentence _("This is done probably to have the smallest network load possible and decrease clone time."_)